core/src/main/java/io/opencensus/stats/ViewData.java

```diff
@@ -53,7 +54,9 @@
 
   /** Constructs a new {@link ViewData}. */
   public static ViewData create(
-      View view, Map<List<TagValue>, List<AggregationData>> map, final WindowData windowData) {
+      View view,
+      Map<? extends List<? extends Object>, List<AggregationData>> map,
```

Comment:
I don't understand it very well, but I think that `List<String>` is treated as a subtype of `List<? extends Object>` (https://stackoverflow.com/a/22806587), so we also need to allow subtypes of the list.  Without the first `? extends`, the second `? extends` seemed to have no effect, and the method wouldn't accept a `List<TagValueString>`.

Another reason to add the first `? extends` is to allow list subclasses, such as ArrayList.